modules/ec2_capacity_provider/asg.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_autoscaling_group" "asg" {
```

Comment:
Used to be a problem describing ASGs in terraform, but not anymore: https://github.com/hashicorp/terraform/issues/1552